external/fv3fit/tests/test_sklearn_wrapper.py

```diff
@@ -210,3 +211,55 @@ def test_predict_returns_unstacked_dims():
     assert len(input_data.dims) > 2
     prediction = wrapper.predict(input_data)
     assert prediction.dims == input_data.dims
+
+
+def test_SklearnWrapper_fit_predict_with_clipped_input_data():
+    nz = 5
+    model = _RegressorEnsemble(
+        base_regressor=DummyRegressor(strategy="constant", constant=np.arange(nz)),
+        n_jobs=1,
+    )
+    wrapper = SklearnWrapper(
+        input_variables=["a", "b"],
+        output_variables=["c"],
+        model=model,
+        packer_config=PackerConfig({"a": {"z": SliceConfig(2, None)}}),
+    )
+
+    dims = ["x", "y", "z"]
+    shape = (2, 2, nz)
+    arr = np.arange(np.prod(shape)).reshape(shape)
+    input_data = xr.Dataset(
+        {"a": (dims, arr), "b": (dims[:-1], arr[:, :, 0]), "c": (dims, arr + 1)}
+    )
+    wrapper.fit([input_data])
+    wrapper.predict(input_data)
+
+
+# clipping for ML outputs is not yet implemented
+def test_SklearnWrapper_fit_with_clipped_output_data():
```

Comment:
What does this comment mean? It looks like it's implemented based on this test existing?

Should the test be named `fit_predict` for consistency with the test above?

Reply:
Okay I slimmed down the test to just check for raising NotImplementedError. There was some crud in there from earlier work where I was trying to implement for outputs too.